.github/workflows/build-deploy.yml

```diff
@@ -0,0 +1,52 @@
+name: build-deploy-bay-images
+run-name: Build and deploy Bay images
+on:
+  schedule:
+    - cron: '23 20 * * 0'
+  workflow_dispatch:
```

Comment:
This event allows for workflows to be triggered [manually via the GitHub UI](https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/). The UI feature is only visible if the workflow file is on the default branch, so we won't see it until this PR is merged.